.github/workflows/build-tfm.yml

```diff
@@ -0,0 +1,29 @@
+name: Build-TFM
+
+on:
+  push:
+    branches:
+        - '**'
+  pull_request:
+    branches:
+        - '**'
+
+jobs:
+  build-tfm:
+    runs-on: ubuntu-latest
+    container: ghcr.io/armmbed/mbed-os-env:master-latest
+    strategy:
+      matrix:
+        board: [ARM_MUSCA_S1, ARM_MUSCA_B1, NU_M2354]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          apt-get update -y
+          apt-get install -y srecord
+          mbed-tools deploy
+          pip install -r mbed-os/requirements.txt
+
+      - name: Build TF-M and all tests
+        run: python3 test_psa_target.py -t GNUARM -m ${{ matrix.board }} --cli=2 -b
```

Comment:
Please test both --cli=2 and --cli=1 here, and remove .circleci from this repo.

Reply:
Could be another item in the matrix?

Reply:
Good point. I'll add it to the matrix.